queries.sql

```diff
@@ -8,5 +8,47 @@ SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE name NOT IN('Gabumon');
 
+/*transaction queries*/
+BEGIN; 
+UPDATE animals SET species='unspecified';
+ROLLBACK;
 
+BEGIN;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon'
+WHERE species IS NULL;
+COMMIT;
```

Comment:
please as required you are asked to check if the changes were made before and after the rollback